src/descriptors/power_system_structs.json

```diff
@@ -1486,7 +1486,7 @@
         "name": "contributingdevices",
         "comment": "devices from which the product can be procured",
         "null_value": "[ThermalStandard(nothing)]",
-        "data_type": "Vector{Device}"
+        "data_type": "Vector{<:Device}"
```

Comment:
The serialization test fails because of this change. I'd like to discuss why you made it.